### ISSUE TITLE
Implement reversal of migration as a command line option

### ISSUE THREAD

**Issue by krafts** [comeara/pillar#29]
I see the class and code exists for reversing the migration. But the cli option is not exposed so the migration can reversed. A required field would be date, I suppose in this case.

**Comment by ccronca** [comment]
I think, ff timestamp parameter is set before now, migrations are applied in reverse order.
https://github.com/comeara/pillar/blob/master/src/main/scala/com/chrisomeara/pillar/CassandraMigrator.scala#L48-L52

**Comment by comeara** [comment]
Yes, that is the case. Thank you for pointing this out. I've left this open as a task for myself to update the documentation to make this clearer.
